tests/DwitTech.AccountService.Core.Tests/Services/UserServiceTest.cs

```diff
@@ -229,41 +230,6 @@ public async Task ChangePasswordAsync_InvalidCurrentPassword_ThrowsArgumentExcep
         }
 
 
-        [Fact]
-        public async Task ChangePasswordAsync_IdenticalPasswords_ThrowsNullArgumentException()
```

Comment:
In the event that the condition I spoke of earlier is replaced, this test will have to be replaced too.